### PR TITLE
Mb ts vardict

### PR DESCRIPTION
**Pull request by migbro**
Updated to balance interval lists, so that they start and end between regions with N's or chromosomes

### CODE REVIEW

File: dev/custom_split.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```
Comment (yuankunzhu) [inline dev/custom_split.py]: it'll be great if you could have an brief usage/function description in comment blocks in terms of what does this script do.

File: tools/vardictjava.cwl [hunk 2]
```diff
@@ -5,24 +5,28 @@ requirements:
   - class: ShellCommandRequirement
   - class: InlineJavascriptRequirement
   - class: ResourceRequirement
-    ramMin: 18
-    coresMin: 4
+    ramMin: ${return inputs.ram * 1000}
+    coresMin: $(inputs.cpus)
   - class: DockerRequirement
-    dockerPull: 'kfdrc/vardict:1.5.8'
+    dockerPull: 'migbro/vardict:1.7.0'
```
Comment (yuankunzhu) [inline tools/vardictjava.cwl]: can we move this image to `kfdrc` docker hub?

Comment (migbro) [reply tools/vardictjava.cwl]: ah! good catch!

File: dev/custom_split.py (hunk 1 above)
Comment (migbro) [reply dev/custom_split.py]: Sure, it was used as the basis for the tool, i was afraid of actually commenting in the tool itself, but can add comments here

Comment (migbro) [reply dev/custom_split.py]: Ok, added to script and added doc description to tool

**Pull request merged**